tests/plugins/balloontoolbar/manual/dropdownhover.md

```diff
@@ -0,0 +1,12 @@
+@bender-ui: collapsed
+@bender-tags: 4.10.2, bug, 1268, balloontoolbar
+@bender-ckeditor-plugins: wysiwygarea,toolbar,basicstyles,balloontoolbar,sourcearea,stylescombo
+
+## For each editor
+
+1. Click on bold text two times.
```

Comment:
There should be also steps for the bug that I reported during the review, so the one with refreshing state after applying styles.